### PR TITLE
Added template_ext = ('.json') to databricks operators #18925

### LINKED ISSUE #18925: Add template_ext = ('.json')  to databricks operators
### Description

Add template_ext = ('.json') in databricks operators. I will improve the debug and the way to organize parameters.

### Use case/motivation

Bigger parameters on file. Easier to debug and maintain.

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by utkarsharma2**
Closes: #18925

### CODE REVIEW

File: airflow/providers/databricks/operators/databricks.py [hunk 1]
```diff
@@ -246,6 +246,7 @@ class DatabricksSubmitRunOperator(BaseOperator):
 
     # Used in airflow.models.BaseOperator
     template_fields: Sequence[str] = ('json',)
+    template_ext: Sequence[str] = ('json',)
```
Comment (potiuk) [inline airflow/providers/databricks/operators/databricks.py]: You should add it as ".json" not "json"

**Review by potiuk — CHANGES_REQUESTED** [review]
Wrong extension specification

**Comment by potiuk** [comment]
I see that you closed a ".json" change? what was the reason?

File: airflow/providers/databricks/operators/databricks.py (hunk 1 above)
Comment (utkarsharma2) [reply airflow/providers/databricks/operators/databricks.py]: @potiuk , I'll update PR with a '.json' extension.

**Review by uranusjr — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**